### DISCUSSION TITLE
Vanilla Contributions

### DISCUSSION THREAD

**Discussion by bitnom** [ionic-team/capacitor#3259] [General]
I'm always looking for a new framework that will allow cross-platform development via a shared codebase. So far, I think  Capacitor comes the closest. I know Javascript (And several other languages). One thing I've never done is become interested in programming frameworks (As opposed to build frameworks). For this reason, I don't know ionic, react, angular, etc.

I'd like to contribute to Capacitor from the perspective and benefit of programmers who don't want to use a programming framework. Perhaps I'm naive in my approach but I feel like if I learn one JS framework, I must learn them all and there's just no time. I know too many languages as it is. To this end, let's be mindful of the vanilla programmer, especially in documentation. If that can be done, I will have the foundation to begin to contribute. I only need to brush up on my TypeScript, which is pretty easy.

**Comment by imhoffd** [comment]
Agreed, we want our docs to be framework-agnostic as possible. Capacitor itself will never require a particular JS framework, but using a modern bundler/module system will improve the experience.

@TensorTom Would you be able to join the [Capacitor Contributor slack](https://getcapacitor.herokuapp.com/)?
